frontend/src/model/accessRequests/AccessRequestDisplay.tsx

```diff
@@ -16,9 +16,10 @@ import ReviewDisplay from '../reviews/ReviewDisplay'
 
 type AccessRequestDisplayProps = {
   accessRequest: AccessRequestInterface
+  displayReviewBanner?: boolean
 }
 
-export default function AccessRequestDisplay({ accessRequest }: AccessRequestDisplayProps) {
+export default function AccessRequestDisplay({ accessRequest, displayReviewBanner = true }: AccessRequestDisplayProps) {
```

Comment:
Usually optional boolean props default to false. Consider reversing the logic and renaming to `hideReviewBanner`. That way when hiding the banner you just need to pass in `hideReviewBanner` instead of `displayReviewBanner={false}`.